prusti-tests/tests/verify_overflow/pass/competitive_programming/dp/SuperSale_10130.rs

```diff
@@ -0,0 +1,268 @@
+// compile-flags: -Zdisable_more_complete_exhale
+// https://onlinejudge.org/external/101/10130.pdf
+use prusti_contracts::*;
+
+use prusti_contracts::*;
+
```

Comment:
```suggestion
```

Reply:
I don't understand why should it be removed. Removing it gives errors in the editor.

Reply:
The `use` is there twice, my suggestion was just to remove the duplicate.

Reply:
Done.